### PR TITLE
Readme adjustments

### PR DESCRIPTION
**Pull request by camilamacedo86**
- remove pipeline status since it does no work
- add a badge for the license and go report
- move health status images for after title/image (keep the same layout of org)
- organize current content by adding the run-local to start it locally near of the installation which shows make more sense.
- Organize alias for links at the end
- Add missing default sections

Click [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/58a2de110e90ef42f10f0f8ed2db9afefabbf124/README.md) to verify these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,15 +24,40 @@ This project does not:
 * Replace [Helm](https://github.com/kubernetes/helm)
 * Turn Kubernetes into a [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service)
 
+## Prerequisites
+
+- [git][git_tool]
+- [go][go_tool] version v1.12+.
+- [mercurial][mercurial_tool] version 3.9+
```
Comment (ecordell) [inline README.md]: Why is mercurial a requirement? should it be listed as an alternative for git?

Comment (camilamacedo86) [reply README.md]: 👍 good spot. Thank you for your time and review. (removed :-))

**Comment by ecordell** [comment]
/lgtm
/approve

Thanks @camilamacedo86 ! Sorry that we let your other PR languish 😢

**Comment by camilamacedo86** [comment]
Hi @ecordell,

It is rebased with the master, could we merge it now?

**Comment by ecordell** [comment]
doc-only - merging without tests.

**Pull request merged**